src/components/organisms/user-search/user-search.view.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import Loading from 'components/atoms/loading';
+import Error from 'components/atoms/error';
+import UserResult from 'components/molecules/user-result';
+import OrganizationResult from 'components/molecules/organization-result';
+
+import {
+  UserSearchContainer,
+  FormContainer,
+  FormErrorContainer,
+} from './user-search.styles';
+
+const UserSearch = ({
+  results,
+  totalCount,
+  hasPrevPage,
+  gotoPrevPage,
+  hasNextPage,
+  gotoNextPage,
+  setSearchQuery,
+  loading,
+  error,
+}) => {
+  const { register, handleSubmit, errors } = useForm();
+  const onSubmit = ({ searchUsers }) => setSearchQuery(searchUsers);
+
+  return (
+    <UserSearchContainer>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FormContainer>
+          <input
+            type="text"
+            placeholder="search field"
+            name="searchUsers"
+            defaultValue={null}
+            ref={register({ required: true })}
+          />
+
+          <input type="submit" />
```

Comment:
This is super ugly. Definitely needs to be styled so the button is closer to the input and give it some more elegance.